R/standardize.models.R

```diff
@@ -471,8 +478,48 @@ standardize.biglm <- standardize.wbm
 # biglm doesn't regit the model to new data - it ADDs MORE data to the model.
 
 #' @export
-standardize.fixest <- standardize.wbm
-# fixest handles its own environment - so we can't update
+# Almost the same as `standardize.default()` but we pass `use_calling_env` in
+# update().
+standardize.fixest <- function(
+  x,
+  robust = FALSE,
+  two_sd = FALSE,
+  weights = TRUE,
+  verbose = TRUE,
+  include_response = TRUE,
+  ...
+) {
+  if (!insight::is_model(x)) {
```

Comment:
This and the next line should probably be moved into the main function, not only for fixest.

Reply:
Ah indeed, thanks, I moved it.